pkg/armrpc/asyncoperation/worker/worker.go

```diff
@@ -179,8 +179,7 @@ func (w *AsyncRequestProcessWorker) Start(ctx context.Context) error {
 
 			if msgreq.DequeueCount > w.options.MaxOperationRetryCount {
 				errMsg := fmt.Sprintf("exceeded max retry count to process async operation message: %d", msgreq.DequeueCount)
-				opLogger.Error(nil, errMsg)
```

Comment:
when we have an async op, the traceID should have been propagated, and this line should have printed the log along with traceID, was it not happening? trying to understand the issue.